godtools/App/Features/Tools/SwiftUI/FavoritingToolBannerView/ViewModel/BaseFavoritingToolBannerViewModel.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  BaseFavoritingToolBannerViewModel.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 5/4/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import Foundation
+
+protocol FavoritingToolBannerDelegate: AnyObject {
```

Comment:
Hey @rachaelblue with the closeTapped() delegate I think it all needs to be handled either in the Base or in the FavoritingToolBannerViewModel subclass.   So either Base knows nothing of the delegate and its strictly part of the FavoritingToolBannerViewModel or all delegate logic is handled in the Base.  I'm leaning more towards moving it all into the FavoritingToolBannerViewModel then leave it up to subclasses to define how they want to handle when closeTapped() is called.  Thoughts?